packages/kernel-6.12/kernel-6.12.spec

```
@@ -0,0 +1,1395 @@
+sed -e 's|PREFIX|%{_cross_prefix}|g' %{S:210} \
+  > %{buildroot}%{_cross_unitdir}/"${LOWERPATH}.mount.d"/no-squashfs.conf
+
+%if "%{_cross_arch}" == "x86_64"
+# Add Neuron-related drop-ins to load the module when the hardware is present.
+mkdir -p %{buildroot}%{_cross_unitdir}/sysinit.target.d
+install -p -m 0644 %{S:220} %{buildroot}%{_cross_unitdir}/sysinit.target.d/neuron.conf
+
+mkdir -p %{buildroot}%{_cross_unitdir}/modprobe@neuron.service.d
+install -p -m 0644 %{S:221} %{buildroot}%{_cross_unitdir}/modprobe@neuron.service.d/neuron.conf
+%endif
+
+# Install platform-specific bootconfig snippets.
+install -d %{buildroot}%{_cross_bootconfigdir}
+install -p -m 0644 %{S:300} %{buildroot}%{_cross_bootconfigdir}/05-aws.conf
+install -p -m 0644 %{S:301} %{buildroot}%{_cross_bootconfigdir}/05-vmware.conf
+
+%files
+%license COPYING LICENSES/preferred/GPL-2.0 LICENSES/exceptions/Linux-syscall-note
+%{_cross_attribution_file}
+/boot/vmlinuz
+/boot/config
+
+%files headers
+%dir %{_cross_includedir}/asm
+%dir %{_cross_includedir}/asm-generic
+%dir %{_cross_includedir}/drm
+%dir %{_cross_includedir}/linux
+%dir %{_cross_includedir}/misc
+%dir %{_cross_includedir}/mtd
+%dir %{_cross_includedir}/rdma
+%dir %{_cross_includedir}/regulator
+%dir %{_cross_includedir}/scsi
+%dir %{_cross_includedir}/sound
+%dir %{_cross_includedir}/video
+%dir %{_cross_includedir}/xen
+%{_cross_includedir}/asm/*
+%{_cross_includedir}/asm-generic/*
+%{_cross_includedir}/drm/*
+%{_cross_includedir}/linux/*
+%{_cross_includedir}/misc/*
+%{_cross_includedir}/mtd/*
+%{_cross_includedir}/rdma/*
+%{_cross_includedir}/regulator/*
+%{_cross_includedir}/scsi/*
+%{_cross_includedir}/sound/*
+%{_cross_includedir}/video/*
+%{_cross_includedir}/xen/*
+
+%files devel
+%dir %{_cross_ksrcdir}
+%{_cross_kmoddir}/source
+%{_cross_kmoddir}/build
+
+%files devel-squashed
+%{_cross_datadir}/bottlerocket/kernel-devel.squashfs
+
+%files devel-unpacked
+%{_cross_datadir}/bottlerocket/kernel-devel
+%{_cross_unitdir}/*kernel*devel*.mount.d/no-squashfs.conf
+
+%files archive
+%{_cross_datadir}/bottlerocket/kernel-devel.tar.xz
+
+%files fips
+%{_cross_unitdir}/check-fips-modules.service.d/*.conf
+
+%files bootconfig-aws
+%{_cross_bootconfigdir}/05-aws.conf
+
+%files bootconfig-vmware
+%{_cross_bootconfigdir}/05-vmware.conf
+
+%files modules
+%dir %{_cross_libdir}/modules
+%dir %{_cross_kmoddir}
+%{_cross_kmoddir}/modules.alias
+%{_cross_kmoddir}/modules.alias.bin
+%{_cross_kmoddir}/modules.builtin
+%{_cross_kmoddir}/modules.builtin.alias.bin
+%{_cross_kmoddir}/modules.builtin.bin
+%{_cross_kmoddir}/modules.builtin.modinfo
+%{_cross_kmoddir}/modules.dep
+%{_cross_kmoddir}/modules.dep.bin
+%{_cross_kmoddir}/modules.devname
+%{_cross_kmoddir}/modules.order
+%{_cross_kmoddir}/modules.softdep
+%{_cross_kmoddir}/modules.symbols
+%{_cross_kmoddir}/modules.symbols.bin
+%{_cross_kmoddir}/modules.weakdep
+%{_cross_kmoddir}/System.map
+%if "%{_cross_arch}" == "x86_64"
```

restore this newline:
```suggestion

%if "%{_cross_arch}" == "x86_64"
```